Task 1/Minimum Deletions to be done/Question.md

```diff
@@ -0,0 +1,4 @@
+<h1>Minimum deletions to be done in given array such that every pair sum is a power of 2 ? </h1>

```

Comment:
The equivalent of <h1> tag in markdown (.md) is #

Might want to check it out 👍